Disable attach_pid tests because they are flaky, they are disabled by default in MacOS and Windows anyways.
--- a/tests/debug/runners.py
+++ b/tests/debug/runners.py
@@ -163,7 +163,7 @@ def _attach_common_config(session, target, cwd):
 @_runner
 @contextlib.contextmanager
 def attach_pid(session, target, cwd=None, wait=True):
-    if wait and not sys.platform.startswith("linux"):
+    if wait:
         pytest.skip("https://github.com/microsoft/ptvsd/issues/1926")
 
     log.info("Attaching {0} to {1} by PID.", session, target)
--- a/tests/debugpy/test_attach.py
+++ b/tests/debugpy/test_attach.py
@@ -153,8 +153,7 @@ def test_reattach(pyfile, target, run):
 
 
 @pytest.mark.parametrize("pid_type", ["int", "str"])
-@pytest.mark.skipif(
-    not sys.platform.startswith("linux"),
+@pytest.mark.skip(
     reason="https://github.com/microsoft/debugpy/issues/311",
 )
 @pytest.mark.flaky(retries=2, delay=1)
